editors/vscode/package.json

```diff
@@ -50,12 +50,35 @@
         "title": "Show Trace Output Channel",
         "category": "Oxc"
       }
-    ]
+    ],
+    "configuration": {
```

Comment:
this configuration to get traces from lsp, e.g: 
![image](https://github.com/oxc-project/oxc/assets/17974631/a62c098d-631d-4ae7-ad88-ebf236e5075b)